pkg/storage/db.go

```diff
@@ -295,14 +295,16 @@ func handleRollback(tx *sql.Tx, err error) error {
 	return err
 }
 
-// GetPartitions fetches the created partitions in the database
+// GetPartitions fetches the created partitions and gets each record count in the database
 func (db *DB) GetPartitions(ctx context.Context) ([]domain.Partition, error) {
-	stmt := "SELECT name, created_at, partition_begin, partition_end FROM partitions ORDER BY partition_end DESC"
+	//stmt := `SELECT name, created_at, partition_begin, partition_end
```

Comment:
Nit: These comments lines should be deleted.

Reply:
Will delete that. Thank you